src/MigrationTools.Clients.AzureDevops.ObjectModel/ProcessorEnrichers/TfsNodeStructure.cs

```diff
@@ -468,5 +475,66 @@ private bool CheckIsParentOfSelectedBasePath(string userFriendlyPath)
             return _nodeBasePaths.Where(onePath => !onePath.StartsWith("!"))
                                  .Any(onePath => onePath.StartsWith(userFriendlyPath));
         }
+
+        public List<string> CheckForMissingPaths(List<WorkItemData> workItems, TfsNodeStructureType nodeType)
+        {
+            EntryForProcessorType(null);
+            _targetCommonStructureService.ClearProjectInfoCache();
+
+            string fieldName = "";
+            switch (nodeType)
+             {
+                case TfsNodeStructureType.Iteration:
+                    fieldName = "System.IterationPath";
+                    break;
+                case TfsNodeStructureType.Area:
+                    fieldName = "System.AreaPath";
+                    break;
+
+            }
+            List<string> areaPaths = (from workItem in workItems where workItem.Fields[fieldName].Value.ToString().Contains("\\") select workItem.Fields[fieldName].Value.ToString()).Distinct().ToList();
+            List<string> missingPaths = new List<string>();
+
+            foreach (var areaPath in areaPaths)
+            {
+                var newpath = GetNewNodeName(areaPath, nodeType);
+                var systempath  = GetSystemPath(newpath, nodeType);
+                try
+                {
+                    NodeInfo c = _targetCommonStructureService.GetNodeFromPath(systempath);
```

Comment:
cant you use _ = _target?

Reply:
No it needs to be the commonstructureservice. _target is not available here